htdocs/compta/prelevement/create.php

```diff
@@ -443,18 +488,28 @@
 	print '</tr>';
 
 	if ($num) {
-		require_once DOL_DOCUMENT_ROOT.'/societe/class/companybankaccount.class.php';
+		if (!GETPOSTISSET('sourcetype')) {
+			require_once DOL_DOCUMENT_ROOT.'/societe/class/companybankaccount.class.php';
+		} else {
+			require_once DOL_DOCUMENT_ROOT.'/user/class/userbankaccount.class.php';
+			require_once DOL_DOCUMENT_ROOT.'/salaries/class/salary.class.php';
+		}
 
 		while ($i < $num && $i < $limit) {
 			$obj = $db->fetch_object($resql);
+			if (!GETPOSTISSET('sourcetype') || empty($type)) {
```

Comment:
A test   $sourcetype != 'salary'   should be enough

Reply:
yep i forgot it here also